### PR TITLE
Adding name, description, sharing options to create and update dataset

### PR DESCRIPTION
**Pull request by calfonso**

### CODE REVIEW

**Comment by kjaym** [comment]
@calfonso Can you bump the version of this library in setup.py. Then add that version to the cli dependency.

File: citrination_client/client.py [hunk 1]
```diff
@@ -119,7 +119,7 @@ def _get_content_from_url(self, url):
 
         :param url: The URL to make the GET request to
         :return: The response object, or an error message object if the request failed
-        """        
+        """
         print(str(url))
```
Comment (kjaym) [inline citrination_client/client.py]: why are you printing the url here?

File: citrination_client/client.py [hunk 2]
```diff
@@ -173,6 +183,33 @@ def _get_create_data_set_url(self):
         """
         return self.api_url+'/data_sets/create_dataset'
 
+    def update_data_set(self, data_set_id, name=None, description=None, share=None):
+        """
+        Update a data set.
+        :param data_set_id:
+        :param name: name of the dataset
+        :param description: description for the dataset
+        :param share: share the dataset with everyone on the site. Valid values are '1' or '0'.
```
Comment (kjaym) [inline citrination_client/client.py]: Can you be very explicit here that 1 means that it is public and 0 means that it is not.

File: citrination_client/client.py (hunk 1 above)
Comment (calfonso) [inline citrination_client/client.py]: I didn't add this. but happy to remove it.

**Pull request merged**